### PR TITLE
Docker 2

### PR DESCRIPTION
**Pull request by kristojorg**
This should fix the previous issues we had with the docker PR

### CODE REVIEW

**Comment by kristojorg** [comment]
Ok I've built the docker image and run it using this branch (and the readme from this branch). Everything seems to be working.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,27 +1,44 @@
 ## CHANGELOG
 
+### 2.1.0
```
Comment (EdwinGuzman) [inline CHANGELOG.md:3]: Since you want to merge this into `beta` first before releasing 2.0.0, then I think this can just be part of 2.0.0 too.

**Comment by kristojorg** [comment]
Okay I made those changes. Once this is merge, I'll update the changelog in 2.0.0

**Comment by EdwinGuzman** [comment]
Can you merge the PR or should I?

**Pull request merged**